navigator_helpers/tasks/evaluation/create_pdf_report.ipynb

```diff
@@ -0,0 +1,826 @@
+    "            if isinstance(value, dict):\n",
+    "                if 'text_diversity_index' in value:\n",
+    "                    text_diversity_scores.append(value['text_diversity_index'])\n",
+    "                if 'gini_simpson_index' in value:\n",
+    "                    gini_scores.append(value['gini_simpson_index'])\n",
+    "    \n",
+    "    avg_text_diversity = sum(text_diversity_scores) / len(text_diversity_scores) if text_diversity_scores else 0\n",
+    "    avg_gini_index = sum(gini_scores) / len(gini_scores) if gini_scores else 0\n",
+    "\n",
+    "    story.append(Paragraph(\"Data Quality Report\", styles['Title']))\n",
+    "    story.append(Spacer(1, 0.2*inch))\n",
+    "\n",
+    "    story.append(Paragraph(\"Key Metrics\", styles['Heading1']))\n",
+    "    \n",
+    "    unique_rows_chart = create_gauge_chart(int(data['row_uniqueness']['percent_unique']))\n",
+    "    semantically_unique_rows_chart = create_gauge_chart(int(data['row_uniqueness']['percent_semantically_unique']))\n",
+    "    text_diversity_chart = create_gauge_chart(int(avg_text_diversity * 100))\n",
+    "    gini_simpson_chart = create_gauge_chart(int(avg_gini_index * 100))\n",
+    "\n",
+    "    unique_rows_title = Paragraph(\"Unique Rows\", styles['ChartTitle'])\n",
+    "    semantically_unique_rows_title = Paragraph(\"Semantically Unique Rows\", styles['ChartTitle'])\n",
+    "    text_diversity_title = Paragraph(\"Text Diversity\", styles['ChartTitle'])\n",
+    "    gini_simpson_title = Paragraph(\"Gini-Simpson Index\", styles['ChartTitle'])\n",
+    "\n",
+    "    chart_table = Table([\n",
+    "        [unique_rows_title, semantically_unique_rows_title, text_diversity_title, gini_simpson_title],\n",
+    "        [unique_rows_chart, semantically_unique_rows_chart, text_diversity_chart, gini_simpson_chart]\n",
+    "    ])\n",
+    "    chart_table_style = TableStyle([\n",
+    "        ('ALIGN', (0, 0), (-1, -1), 'CENTER'),\n",
+    "        ('VALIGN', (0, 0), (-1, -1), 'MIDDLE'),\n",
+    "        ('BOTTOMPADDING', (0, 0), (-1, 0), 3), \n",
+    "        ('TOPPADDING', (0, 1), (-1, -1), 3),\n",
+    "    ])\n",
+    "    chart_table.setStyle(chart_table_style)\n",
+    "\n",
+    "    story.append(chart_table)\n",
+    "    story.append(Paragraph(\"• Unique Rows: percentage of rows that are unique in the dataset.\", styles['BodyText']))\n",
+    "    story.append(Paragraph(\"• Semantically Unique Rows: percentage of rows that are semantically unique, based on TF-IDF\", styles['BodyText']))\n",
+    "    story.append(Paragraph(\"• Text Diversity: avg text diversity across all text fields, with higher values indicating more diverse content.\", styles['BodyText']))\n",
+    "    story.append(Paragraph(\"• Gini-Simpson Index: probability that two randomly selected entities from the dataset will be different, with higher values indicating greater diversity.\", styles['BodyText']))\n",
```

Comment:
Suggested edit:
Gini-Simpson Index: average diversity index across all categorical fields, defined as the probability that two randomly selected categories from the field will be different, with higher values indicating greater diversity.

(I think there's another description down below where we should similarly point out that this is only for categorical fields)